packages/valory/skills/liquidity_trader_abci/models.py

```diff
@@ -107,4 +117,29 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.chain_to_chain_id_mapping = json.loads(
             self._ensure("chain_to_chain_id_mapping", kwargs, str)
         )
-        super().__init__(*args, **kwargs)
```

Comment:
here

Reply:
Well spotted!

Reply:
Thanks for catching that! I've corrected the mistake @0xArdi